Added so equals and hashcode is generated in models. (#920)
quarkiverse/quarkus-openapi-generator: docs/modules/ROOT/pages/includes/equals-hashcode.adoc

> `Hashcode` and `equals` are by default generated. It adds all the variables in the model
> to the model to the methods.

Reviewer: ```suggestion
`Hashcode` and `equals` are by default generated. It adds all the variables in the model to the model to the methods.
```
I am not sure if I understood this sentence. 🤔

Author: In my head it was 100% clear when I pushed it :D I will try and refrase it! :)

Author: I rewrote the documentation, I hope this ones better.

> `hashcode` and `equals` are by default generated to all models. `hashcode` and `equals` is based on all the variables
> in the model and can't be customized.

Reviewer 2: ```suggestion
By default, `hashcode` and `equals` methods are automatically generated for all models. These methods are based on all the variables in the model and can not be customized.
```

Reviewer 2: Nice PR!
